crates/re_space_view_text_log/src/space_view_class.rs

```diff
@@ -402,16 +406,60 @@ fn table_ui(
 
                 // body
                 row.col(|ui| {
-                    let mut text = egui::RichText::new(entry.body.as_str());
+                    if state.selectable {
+                        // Use a &str with TextEdit so body is selectable but immutable.
+                        let entry_text_rows = entry.num_body_lines;
+                        let mut binding = entry.body.as_str();
+                        let mut text = egui::TextEdit::multiline(&mut binding)
+                            .min_size([0.0, 0.0].into())
+                            .desired_rows(entry_text_rows)
+                            .desired_width(f32::INFINITY);
+                        // Font information for LayoutJob, based on default_layouter in `egui::TextEdit::show_contents`
+                        let font_id = if state.monospace {
+                            egui::TextStyle::Monospace.into()
+                        } else {
+                            egui::FontSelection::default()
+                        }
+                        .resolve(ui.style());
+                        let text_color = if let Some(color) = entry.color {
+                            color.into()
+                        } else {
+                            // TODO(lupickup): Understand why TextEdit text color is white instead of gray like the RichText version.
+                            ui.visuals()
+                                .override_text_color
+                                .unwrap_or_else(|| ui.visuals().widgets.inactive.text_color())
```

Comment:
`inactive.text_color` is the text color of an interactive widget (that is not currently being interacted with). That's why you get white text: https://docs.rs/egui/latest/egui/style/struct.Widgets.html#structfield.inactive

Use `noninteractive` instead (we don't count text selection as interaction)